src/commands/functions-env-set.ts

```diff
@@ -56,6 +58,9 @@ export default new Command("functions:env:set [values...]")
     }
 
     const envs = await fenv.setEnvs(projectId, setEnvs);
-    logger.info(fenv.formatEnv(envs));
+    logger.info(fenv.formatEnv(envs) + "\n");
+    utils.logWarning(
+      "Please deploy your functions for the change to take effect by running firebase deploy --only functions"
```

Comment:
Nit: We should bold the command here, like we do in https://github.com/firebase/firebase-tools/blob/66447909ab6f2defd04b257b48193c70d778c7ea/src/getProjectId.js#L34